### PR TITLE
Minor MISRA fixes.

### PR DESCRIPTION
**Pull request by lundinc2**
This PR resolves recent regressions for MISRA.

### CODE REVIEW

**Review by dachalco — APPROVED** [review]

File: source/portable/mbedtls/core_pkcs11_mbedtls.c [hunk 1]
```diff
@@ -4840,10 +4839,10 @@ CK_DECLARE_FUNCTION( CK_RV, C_Verify )( CK_SESSION_HANDLE hSession,
         }
         else if( pxSessionObj->xOperationVerifyMechanism == CKM_AES_CMAC )
         {
-            if( ulSignatureLen != MBEDTLS_AES_BLOCK_SIZE )
+            if( ulSignatureLen != PKCS11_AES_CMAC_MIN_SIZE )
```
Comment (gkwicker) [inline source/portable/mbedtls/core_pkcs11_mbedtls.c:4842]: Is this change just for MISRA or is this a bugfix?

**Review by gkwicker — COMMENTED** [review]
One question

File: source/portable/mbedtls/core_pkcs11_mbedtls.c (hunk 1 above)
Comment (lundinc2) [reply source/portable/mbedtls/core_pkcs11_mbedtls.c:4842]: This is just for MISRA, the existing is a macro defined by mbedtls which is `16` the new is a macro we already defined for the AES CMAC block length

Comment (lundinc2) [reply source/portable/mbedtls/core_pkcs11_mbedtls.c:4842]: Our's is `16UL` so MISRA doesn't complain.

**Review by gkwicker — APPROVED** [review]

**Pull request merged**